docs/examples/tutorial/string/assignment.pyx

```diff
@@ -0,0 +1,12 @@
+from libc.stdlib cimport free
+from c_func cimport c_call_returning_a_c_string
+
+
+def main():
+    cdef char* c_string = c_call_returning_a_c_string()
+    cdef bytes py_string = c_string
+
+    # A type cast to `object` or `bytes` will do the same thing:
+    py_string = <bytes> c_string
+
+    free(c_string)

```

Comment:
This definitely needs a try-finally, because the object creation might fail with `MemoryError`.
And since that adds even more complexity to an originally very simple piece of example code, I'd rather leave this specific 2-line snippet untouched.